config.yml

```diff
@@ -213,6 +213,9 @@ elrond:
 filecoin:
   api: https://api.filscan.io:8700/rpc/v1
   explorer: https://filfox.info
+  
+oasis:
+  api: https://oasis.atlas.zondax.ch
```

Comment:
can we use this endpoint in production?

Reply:
Please use it for testing for now

> do you have sample address to test txs api?

Here is an example address: `oasis1qpqu7cfadhayekanmrtk8t3v4cx9483gmqu7h6gn`
https://www.oasisscan.com/accounts/detail/oasis1qzt0cma7p9yyuqdz56kkg0t7g9al96glm50aufmy